src/app/js/formats/vega-lite/VegaLiteView.js

```diff
@@ -20,7 +20,7 @@ class VegaLiteView extends Component {
         let spec;
 
         try {
-            spec = JSON.parse(specTemplate);
+            spec = this.retroCompatibilityCheck(JSON.parse(specTemplate), data);
```

Comment:
The `data` parameter can be remove for now, but is possible to be used for fixing area charts